### PR TITLE
[New] Object Storage Use Cases

### PR DESCRIPTION
**Pull request by andystevensname**
This will need be socialized with the OBJ team before the tech/copy edit phase.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 118d750e0209a07aeb5b96a33ef58af13a124221

https://deploy-preview-2680--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit d7ae4ab428389163af9248e648029dca3ca822f4

https://deploy-preview-2680--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Working on tech edit of this

File: docs/platform/object-storage/object-storage-use-cases/index.md [hunk 1]
```diff
@@ -55,14 +55,17 @@ Big Data typically describes data sets that are so large and so diverse that it
...
 
 Similarly, Object Storage has benefits over [tape storage](https://en.wikipedia.org/wiki/Tape_drive). Tape storage is frequently used for archival purposes, but the read times that come with tape storage are many times more than what you'll find with Object Storage. Special considerations have to be made when transferring tape drive data, such as the ability to ship drives safely across a country. With Object Storage, this data is available through HTTP from anywhere in the world.
 
+{{< note >}}
+The outbound data transfer for Linode Object Storage is part of your Linode account's [total transfer pool](/docs/platform/billing-and-support/network-transfer-quota/), which will reduce or completely eliminate transfer costs for object storage if you are running Linode instances too. When you have expended your allotted transfer pool, you will be billed at a rate of $0.02 per GB for outbound transfers.
+{{< /note >}}
```
Comment (rsyracuse) [inline docs/platform/object-storage/object-storage-use-cases/index.md]: Adjusted the information on billing to clarify what you will and won't be billed for. With enough of a transfer pool, you won't be billed for transfer usage at all

**Pull request merged**